roles/satellite-clone/tasks/main.yml

```diff
@@ -1,17 +1,22 @@
 ---
-- include_vars: main.yml
-
 # include pre_install_check
 - include: pre_install_check.yml
   when: run_pre_install_check
 
+- name: "check that variables are updated"
```

Comment:
this can go inside pre_install_check.yml - all the other checks (including variables) are there